tutorials/15-external-samplers/external-samplers.jmd

```diff
@@ -0,0 +1,104 @@
+---
+title: Using External Sampler
+permalink: /tutorials/:name/
+mathjax: true
+weave_options:
+  error : false
+---
+
+# Using External Samplers
+
+Turing currently wraps some of the most popular sampling libraries in the Julia ecosystem.
+These wrappers allow new users to seemlessly sample statistical models without leaving Turing.
+However, these wrappers often lack the latests functionalities implemented on the sampling libraries that more experienced users might want to use.
+Moreover, users might want to use samplers currently not wrapped within Turing.
+
+For these reasons, Turing now allows users the possibility of defining their own sampler and using it to sample a Turing model.
+Let us begin by defining a simple Turing model to showcase the use of external samplers.
+For this demonstration we will use a 10 dimensional Neal's funnel:
+
+```julia
+# Import libraries.
+using Turing, Random, LinearAlgebra
+
+d = 10
+@model function funnel()
+    θ ~ Truncated(Normal(0, 3), -3, 3)
+    z ~ MvNormal(zeros(d - 1), exp(θ) * I)
+    return x ~ MvNormal(z, I)
+end
+```
+
+Let us know sample the model to generate some observations and condition our model on them:
+
+```julia
+Random.seed!(1)
+(; x) = rand(funnel() | (θ=0,))
+model = funnel() | (; x)
+```
+
+Users can now use any sampler algorithm they wish to sample this model as long as it is of the type `AbstractMCMC.AbstractSampler`.
```

Comment:
```suggestion
Users can use any sampler algorithm they wish to sample this model as long as it is of the type `AbstractMCMC.AbstractSampler`.
```

Generally speaking I don't think we want to refer to "the time of writing" (unless we're describing behavior that might be changing for some reason or a bug), since users will generally be reading this without any regard for when it was written (other than assuming that it is up to date). If things change, we should change the docs; hence referene to time seems unnecessary.